--- basket/src/basket.cpp	2007-01-31 09:53:47.000000000 +0100
+++ basket/src/basket.cpp	2007-01-31 09:54:40.000000000 +0100
@@ -1106,7 +1106,7 @@ bool Basket::save()
 
 void Basket::load()
 {
-	kdDebug() << "basket is loading" << endl ;
+	fprintf(stderr,"basket is loading\n");
 	// Load only once:
 	if (m_loadingLaunched)
 		return;
@@ -3781,13 +3781,13 @@ void Basket::closeEditorDelayed()
 
 bool Basket::closeEditor()
 {
-	fprintf(stderr,"Editor is being closed\n");
+	kdDebug() << "Editor is being closed" << endl ;
 
 	if (!isDuringEdit())
 		return true;
 
 	if (m_doNotCloseEditor) {
-		fprintf(stderr,"But close canceled\n");
+		kdDebug() << "But close canceled" << endl ;
 		return true;
 	}
 
